Ports/ffmpeg/package.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=ffmpeg
+version=4.4
+useconfigure=true
+files="https://ffmpeg.org/releases/ffmpeg-${version}.tar.gz ffmpeg-${version}.tar.gz a4abede145de22eaf233baa1726e38e137f5698d9edd61b5763cd02b883f3c7c"
+auth_type="sha256"
+makeopts="-j$(nproc)"
```

Comment:
This seems gratuitous, I don't think a random port should force parallelism on the build system.

Reply:
Good call. I got that line from the lua package script. (which is what I originally copied and modified because I'm familiar with lua)

During testing I compiled ffmpeg without parallelism (to make debugging easier) and I will warn that it did take a non-negligible amount of time to compile.

I'll take that line out.

Reply:
Ok. I took care of it!